Assets/LeapMotion/Scripts/LeapImageRetriever.cs

```diff
@@ -37,18 +38,14 @@ public class LeapImageRetriever : MonoBehaviour {
   [FormerlySerializedAs("gammaCorrection")]
   private float _gammaCorrection = 1.0f;
 
-  private int _missedImages = 0;
   private EyeTextureData _eyeTextureData = new EyeTextureData();
 
   //Image that we have requested from the service.  Are requested in Update and retrieved in OnPreRender
   protected Image _requestedImage = new Image();
-
-  //Debug vars for image retrieval
-  protected int _requestedImages = 0;
-  protected int _lateImages = 0;
-  protected int _superLateImages = 0;
-  protected float _startTime = 0;
-  protected float _lastTime = 0;
+  [SerializeField]
+  protected long ImageTimeout = 9000; //microseconds
```

Comment:
It is used in OnPreRender (329) in place of the 9000 microsecond hardcoded timeout value.